### DISCUSSION TITLE
Day2 turtlesim 질문입니다.

### DISCUSSION THREAD

**Discussion by pol0507** [Road-Balance/gcamp_ros2_basic#35] [Q&A]
![사진1](https://user-images.githubusercontent.com/109154324/178557380-6824f20b-f83c-4aa9-903b-cef8654d362f.JPG)
![사진2](https://user-images.githubusercontent.com/109154324/178557398-0b2ad214-455a-45da-aa43-967dbc74fb35.JPG)

Day2_Node_Package_Launch의 5페이지 turtlesim 실행하는 과정에서 
일단 turtlesim 화면이 보이긴 하지만 
터미널2로 거북이를 조종하려고 하면 제자리에서 멈춘 상태로 거북이의 방향만 빙글빙글 돌며
사진 2와 같은 문구들이 뜹니다.

**Comment by kimsooyoung** [comment]
키보드가 잘못 눌리고 있지는 않았는지 확인 부탁드립니다.

해당 터미널 로그는 무언가 키가 눌렸을 때 등장하는 부분입니다.

**Reply by kimsooyoung** [reply]
해결 되셨을까요?
